project_euler/problem_03/sol04.py

```diff
@@ -31,21 +31,12 @@ def solution(n):
     Traceback (most recent call last):
         ...
     ValueError: Parameter n must be greater or equal to one.
-    >>> solution([])
-    Traceback (most recent call last):
-        ...
-    TypeError: Parameter n must be int or passive of cast to int.
-    >>> solution("asd")
-    Traceback (most recent call last):
-        ...
-    TypeError: Parameter n must be int or passive of cast to int.
```

Comment:
Why remove these test?

Reply:
@onlinejudge95 told  me to remove the TypeError exception, so we only pass integer and those tests become irrelevant as there is no need to check for string.

Anyway, I thought the main idea here is the Algorithm itself, and not getting busy with all validation.